ctapipe/image/hillas.py

```diff
@@ -165,6 +165,28 @@ def hillas_parameters(geom, image):
 
         m4_long = np.average(longitudinal ** 4, weights=image)
         kurtosis_long = m4_long / length ** 4
+    
+    """
+    Compute Hillas parameters uncertainties. 
+    Implementation based on W. Wittek. 2002. Image parameters. MAGIC-TDAS internal note 02-03. 
```

Comment:
Please add this to `docs/bibliography.rst` and move this comment into the real docstring of the function

Reply:
Added to `docs/bibliography.rst`